ci-operator/config/openshift/verification-tests/openshift-verification-tests-master__installer-rehearse-4.12.yaml

```diff
@@ -39,7 +39,7 @@ tests:
     cluster_profile: aws-qe
     env:
       BASE_DOMAIN: qe.devcluster.openshift.com
-    workflow: cucushift-installer-rehearse-aws-ipi-ingress-lbtype-classic
+    workflow: cucushift-installer-rehearse-aws-ipi-byo-kms-etcd-encryption
```

Comment:
We will have a followup PR to add the coverage into ci job yaml file, right?

Reply:
yes, for 4.11-4.12